### PR TITLE
Changes for exporting redis only metrics if needed

### PR DESCRIPTION
**Pull request by nipuntalukdar**
Added support for exporting Redis only metrics if required. Default behavior is same as existing behavior, i.e., all Redis metrics including metrics for go runtime and go http handlers are exported. If the exporter is run with "-redis-only-metrics true", then only Redis specific metrices are exported.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -28,6 +29,7 @@ var (
 	logFormat     = flag.String("log-format", "txt", "Log format, valid options are txt and json")
 	showVersion   = flag.Bool("version", false, "Show version information and exit")
 	useCfBindings = flag.Bool("use-cf-bindings", false, "Use Cloud Foundry service bindings")
+	redisOnly     = flag.Bool("redis-only-metrics", false, "Whether to export go runtime metrics also")
```
Comment (oliver006) [inline main.go]: Could you rename this to `redisMetricsOnly`?

File: main.go [hunk 2]
```diff
@@ -86,16 +88,25 @@ func main() {
...
 
 	buildInfo := prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "redis_exporter_build_info",
 		Help: "redis exporter build_info",
 	}, []string{"version", "commit_sha", "build_date", "golang_version"})
-	prometheus.MustRegister(buildInfo)
 	buildInfo.WithLabelValues(VERSION, COMMIT_SHA1, BUILD_DATE, runtime.Version()).Set(1)
 
-	http.Handle(*metricPath, prometheus.Handler())
+	if !*redisOnly {
```
Comment (oliver006) [inline main.go]: Could you switch this and make it 
```
if *redisMetricsOnly {

} else {

}
```

**Comment by nipuntalukdar** [comment]
Made changes as suggested in review comments.

**Comment by oliver006** [comment]
Looks good, thank you!

**Pull request merged**